src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientStatisticsSystem.java

```diff
@@ -0,0 +1,111 @@
+    private NUIManager nuiManager;
+    @In
+    private LocalPlayer localPlayer;
+    @In
+    private ClientGameOverSystem clientGameOverSystem;
+    @In
+    InputSystem inputSystem;
+
+    private static final String NOTIFICATION_ID = "LightAndShadow:firstTime";
+    private int redScore = 0;
+    private int blackScore = 0;
+
+    @Override
+    public void initialise() {
+        statisticsScreen = nuiManager.createScreen("LightAndShadow:statisticsScreen", DeathScreen.class);
+        clientGameOverSystem.addGoalScore(statisticsScreen, "spadesGoalScore");
+        clientGameOverSystem.addGoalScore(statisticsScreen, "heartsGoalScore");
+        clientGameOverSystem.addTeamScore(statisticsScreen, "heartsTeamScore", redScore);
+        clientGameOverSystem.addTeamScore(statisticsScreen, "spadesTeamScore", blackScore);
+
+    }
+    @ReceiveEvent
+    public void onScoreUpdate(ScoreUpdateFromServerEvent event, EntityRef entity) {
+        String team = event.team;
+        if (team.equals(LASUtils.RED_TEAM)) {
+            redScore = event.score;
+        }
+        if (team.equals(LASUtils.BLACK_TEAM)) {
+            blackScore = event.score;
+        }
+    }
+
+    @ReceiveEvent(priority = EventPriority.PRIORITY_CRITICAL)
+    public void onTab(TabButton event, EntityRef entity) {
+        if (event.getState() == ButtonState.DOWN) {
+            if (localPlayer.getClientEntity().equals(entity)) {
+                entity.send(new ExpireNotificationEvent(NOTIFICATION_ID));
+                clientGameOverSystem.addTeamScore(statisticsScreen, "spadesTeamScore", blackScore);
+                clientGameOverSystem.addTeamScore(statisticsScreen, "heartsTeamScore", redScore);
+                clientGameOverSystem.addPlayerStatisticsInfo(statisticsScreen);
+                nuiManager.toggleScreen("LightAndShadow:statisticsScreen");
+                event.consume();
+            }
+        }
+    }
+
+    private String getActivationKey(SimpleUri button) {
+        return inputSystem.getInputsForBindButton(button).stream()
+                .findFirst()
+                .map(Input::getDisplayName)
+                .map(key -> {
+                    if (key.length() == 1) {
+                        // print the key in yellow within a circle
+                        int off = key.charAt(0) - 'A';
+                        char code = (char) (EnclosedAlphanumerics.CIRCLED_LATIN_CAPITAL_LETTER_A + off);
+                        return String.valueOf(code);
+                    } else {
+                        return key;
+                    }
+                })
+                .map(key -> FontColor.getColored(key, Color.yellow))
+                .orElse(FontColor.getColored("n/a", Color.red));
+    }
+
+    @ReceiveEvent
+    public void onLocalPlayerInitialized(LocalPlayerInitializedEvent event, EntityRef entity) {
+            Notification notification = new Notification(NOTIFICATION_ID,
+                    "How to open the Statistics Screen?",
```

Comment:
I would definitely go with something shorter. For the other notifications we used something not so "technical" or "official" or "serious", but rather something remotely funny or that had a kind of "ring" to it...

For this I'd propose 
```suggestion
                    "The Numbers Game",
```